client/dashboard/sites/logs-activity/dataviews/index.tsx

```diff
@@ -63,9 +71,12 @@ function SiteActivityLogsDataViews( {
 			fields={ fields as Field< SiteActivityLog >[] }
 			view={ view }
 			actions={ actions }
+			config={ hasActivityLogsAccess ? undefined : { perPageSizes: [ 20 ] } } // Disable changing perPage if no access
 			search={ false }
 			defaultLayouts={ { table: {} } }
 			onChangeView={ onChangeView }
+			empty={ <p>{ view.search ? __( 'No activity found' ) : __( 'No activities' ) }</p> }
+			children={ hasActivityLogsAccess ? undefined : <DataViews.Layout /> } // showing only the layout when on the free plan.
```

Comment:
Actually I don't quite get this -- why do we want to show the controls on Free plan?